### PR TITLE
Zeta Global SSP: Update endpoints

### PR DESCRIPTION
**Pull request by abermanov-zeta**
Get rid of `shortname` in bidding and user sync endpoints

### CODE REVIEW

File: static/bidder-info/zeta_global_ssp.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-endpoint: https://ssp.disqus.com/bid/prebid-server?sid=GET_SID_FROM_ZETA&shortname=GET_SHORTNAME_FROM_ZETA
+endpoint: https://ssp.disqus.com/bid/prebid-server?sid=GET_SID_FROM_ZETA
```
Comment (onkarvhanumante) [inline static/bidder-info/zeta_global_ssp.yaml:1]: @abermanov-zeta  please update json framework tests to use endpoints defined in bidder-info yaml. This ensures that adapter code is compatible with actual endpoint mentioned in yaml.

https://github.com/prebid/prebid-server/blob/754a0ebcdc8b3ac5666986fc81ea762f1b8e1de9/adapters/zeta_global_ssp/zeta_global_ssp_test.go#L12-L16

Comment (onkarvhanumante) [reply static/bidder-info/zeta_global_ssp.yaml:1]: > @abermanov-zeta please update json framework tests to use endpoints defined in bidder-info yaml. This ensures that adapter code is compatible with actual endpoint mentioned in yaml.
> 
> https://github.com/prebid/prebid-server/blob/754a0ebcdc8b3ac5666986fc81ea762f1b8e1de9/adapters/zeta_global_ssp/zeta_global_ssp_test.go#L12-L16

@abermanov-zeta requesting to address comment to move forward with this PR

**Comment by onkarvhanumante** [comment]
@abermanov-zeta,  prebid server has published major release 2.0. 

Requesting to sync with  `abermanov-zeta:feature/endpoints_update` with latest prebid server master.

File: static/bidder-info/zeta_global_ssp.yaml (hunk 1 above)
Comment (abermanov-zeta) [reply static/bidder-info/zeta_global_ssp.yaml:1]: @onkarvhanumante Sorry for the delay, I was on vacation. Tests have been updated. Thank you

**Review by onkarvhanumante — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**